src/WebJobs.Script.WebHost/Middleware/HostWarmupMiddleware.cs

```diff
@@ -28,6 +30,7 @@ public HostWarmupMiddleware(RequestDelegate next, IScriptWebHostEnvironment webH
             _environment = environment;
             _hostManager = hostManager;
             _logger = logger;
+            _assemblyLocalPath = Path.GetDirectoryName(new Uri(typeof(HostWarmupMiddleware).Assembly.Location).LocalPath);
```

Comment:
Yes - we need the directory where host is running from - when running in Azure it would be SiteExtensions folder - @safihamid  can we change this `GetCurrentDirectory`?

Reply:
I am not sure if it works, plus this whole logic is all happening during placeholder mode so I am not too concerned about it perfwise.